frontend/src/component/feature/FeatureStrategy/FeatureStrategyMenu/FeatureReleasePlanCard/FeatureReleasePlanCard.tsx

```diff
@@ -49,13 +50,12 @@ const StyledCard = styled('div')(({ theme }) => ({
     textAlign: 'left',
     overflow: 'hidden',
     position: 'relative',
-    [`&:hover ${CardContent}, &:focus-within ${CardContent}`]: {
+    '&:hover > div:first-of-type, &:focus-within > div:first-of-type': {
```

Comment:
Right, if I'm reading these right, you want the main card to lose opacity and the buttons to gain opacity, yeah? The selectors here might work, but they're also pretty brittle. Why not use html classes for this? Alternatively, you can apply the styling within the affected components directly.